Entities/Items/Crate/Crate.as

```diff
@@ -88,6 +89,8 @@ void onInit(CBlob@ this)
 	}
 
 	this.getSprite().SetZ(-10.0f);
+	
+	this.Tag("sawed"); //HACK: prevents saws from sawing crates :(
```

Comment:
This kind of logic is/should be handled in Saw.as:156

Reply:
Should I make an exception for crate, or add a "do not saw" tag?

Reply:
A tag like that sounds more sane indeed, maybe for another PR though.